ur_robot_driver/doc/usage_example.md

```diff
@@ -0,0 +1,204 @@
+We assume the use of the URCap in the following.
+
+Before any control of the robot can happen, a program containing the **External Control** program
+node must be running on the robot. You should see the output `Robot ready to receive control
+commands.` in the terminal where you launched the driver. See [this troubleshooting
+entry](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/README.md#i-started-everything-but-i-cannot-control-the-robot)
+if you experience problems in this step.
+
+### Control the robot using the `test_move` script
+Once you've setup the robot for getting controlled by ROS, you can move it around using the
+[`test_move`](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/test_move) script.
+
+Start the driver and an RViz window as before. Then, in a third terminal run
+
+```
+rosrun ur_robot_driver tes_move
+```
+
+This script will guide you through the different trajectory controllers available. For informations
+about the different controllers, please see [TODO: Change once merged #423](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/423).
+
+It will first ask about the controller to be used:
+
+```
+Available trajectory controllers:
+0 (joint-based): scaled_pos_joint_traj_controller
+1 (joint-based): scaled_vel_joint_traj_controller
+2 (joint-based): pos_joint_traj_controller
+3 (joint-based): vel_joint_traj_controller
+4 (joint-based): forward_joint_traj_controller
+5 (Cartesian): pose_based_cartesian_traj_controller
+6 (Cartesian): joint_based_cartesian_traj_controller
+7 (Cartesian): forward_cartesian_traj_controller
+Please choose a controller by entering its number (Enter '0' if you are unsure / don't care): 0
+```
+
+After that, you'll have to confirm that the motions the robot is about to make, can safely be
+executed the way your robot is mounted. If you're running a simulated robot, you can confirm this
+without thinking twice. However, if a real robot is connected, please make sure that the robot is
+actually free of any obstacles inside the space where the motion is taking place. See the images
+below for a visualization of the setpoints:
+
+**Joint-based motions**: (The cable leaving the base to the back on an e-Series robot)
+![joint_based pose 1](resources/usage_example/joint_based_motion_1.png "Pose one of joint-based motion")
+![joint_based pose 2](resources/usage_example/joint_based_motion_2.png "Pose two of joint-based motion")
+![joint_based pose 3](resources/usage_example/joint_based_motion_3.png "Pose three of joint-based motion")
+
+**Cartesian motions**: (The cable leaving the base to the back on an e-Series robot)
+**Please note: Depending on your starting configuration there might also be a valid elbow-down
+configuration for each of the poses!**
+
+![cartesian pose 1](resources/usage_example/cartesian_motion_1.png "Pose one of Cartesian motion")
```

Comment:
See comment above about a single image for this.

Reply:
Resolved in 426ea83